egs/librimix/ConvTasNet/train.py

```diff
@@ -84,24 +84,24 @@ def main(conf):
     )
 
     # Define callbacks
+    callbacks = []
     checkpoint_dir = os.path.join(exp_dir, "checkpoints/")
     checkpoint = ModelCheckpoint(
         checkpoint_dir, monitor="val_loss", mode="min", save_top_k=5, verbose=True
     )
-    early_stopping = False
+    callbacks.append(checkpoint)
     if conf["training"]["early_stop"]:
-        early_stopping = EarlyStopping(monitor="val_loss", patience=30, verbose=True)
+        callbacks.append(EarlyStopping(monitor="val_loss", mode="min", patience=30, verbose=True))
 
     # Don't ask GPU if they are not available.
     gpus = -1 if torch.cuda.is_available() else None
     trainer = pl.Trainer(
         max_epochs=conf["training"]["epochs"],
-        checkpoint_callback=checkpoint,
-        early_stop_callback=early_stopping,
+        callbacks=callbacks,
         default_root_dir=exp_dir,
         gpus=gpus,
         distributed_backend="dp",
```

Comment:
I started to test on LibriMix. I'm letting you know what I've found.

This breaks if you don't have any gpu. A workaround is :

```suggestion
        distributed_backend="distributed_backend",
```
And declare `distributed_backend = "ddp" if torch.cuda.is_available() else None`
I think we should use by default `distributed_backend = "ddp"` instead of `distributed_backend = "dp"` in all recipes